crates/sui-framework/packages/sui-framework/tests/random_tests.move

```diff
@@ -442,8 +456,10 @@ module sui::random_tests {
         let _output = gen.generate_u128_in_range(0, 0xFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF);
         let mut i = 0;
         while (i < 50) {
-            let min = gen.generate_u128();
-            let max = min + (gen.generate_u64() as u128);
+            let (min, max) = (gen.generate_u128(), gen.generate_u128());
```

Comment:
Fixed logic here since this was incorrect and subject to overflow (and the tests were in fact failing for this).